src/phoenix/trace/otel.py

```diff
@@ -291,7 +314,10 @@ def encode(span: Span) -> otlp.Span:
             attributes.pop(key, None)
         elif isinstance(value, Mapping):
             attributes.pop(key, None)
-            attributes.update(_flatten_mapping(value, key))
```

Comment:
Are both of these codepaths needed? Seems like only one should be necessary?

Reply:
we would not know which is json string, unless we try to `json.loads` every string